tools/benchmark/binsize/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+# This file is part of ICU4X. For terms of use, please see the file
+# called LICENSE at the top level of the ICU4X source tree
+# (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+[package]
+name = "icu_benchmark_binsize"
+version = "0.1.0"
+authors = ["The ICU4X Project Developers"]
+edition = "2018"
+
+include = [
+    "src/**/*",
+    "examples/**/*",
+    "Cargo.toml",
+    "README.md"
+]
+
+[dependencies]
+serde_json = "1.0"
```

Comment:
question: Is this dependency still going to be used?

Do you intend to update this PR later to create a Rust struct for each ndjson data point, and want to use `serde_json` to serialize it, rather than hand-writing your own JSON string patterns for `println!()`?

Reply:
I would prefer to rely on something like this rather than relying on stdout. It would be more code though.

Reply:
ad echeran: no, at least not at this time.
ad gregtatum: noted; at this time I'd prefer to keep it shorter.